pages/login.tsx

```diff
@@ -1,7 +1,35 @@
 import * as React from 'react'
+import Layout from '../components/Layout'
+import Card from '../components/Card'
 
 const Login: React.FC = () => {
-  return <h1>Login</h1>
+  return (
+    <Layout>
+      <Card title="Login">
+        <form>
+          <div className="form-group">
+            <input
+              placeholder="Email address or username"
+              type="text"
+              className="form-control form-control-lg font-weight-light mb-3"
+            />
+          </div>
+          <input
+            placeholder="Password"
+            type="text"
+            className="form-control form-control-lg font-weight-light mb-3"
+          />
+          <button
+            type="button"
+            className="btn btn-primary btn-lg btn-block mb-3"
+          >
+            Login to Your Account
+          </button>
+          <a href="">Forgot your password?</a>
```

Comment:
could you wrap this around a `p` tag? Its better for screen readers.

Reply:
are you referring to <a>? wrap the p tag around <a>?

Reply:
> could you wrap this around a `p` tag? Its better for screen readers.

are you referring to wrapping the p tag over a tag?

Reply:
Yes he does `<p><a>link</a></p>`

Reply:
```
import Link from 'next/link'
...
<Link href="/forgotPassword">
  <a>Forgot Password?</a>
</Link>
```